lnrpc/routerrpc/router_server.go

```diff
@@ -491,6 +487,22 @@ func (s *Server) QueryMissionControl(ctx context.Context,
 	return &response, nil
 }
 
+// toRPCPairData marshalls mission control pair data to the rpc struct.
+func toRPCPairData(data *routing.TimedPairResult) *PairData {
+	rpcData := PairData{
+		MinPenalizeAmtSat: int64(
+			data.MinPenalizeAmt.ToSatoshis(),
+		),
+		LastAttemptSuccessful: data.Success,
+	}
+
+	if !data.Timestamp.IsZero() {
+		rpcData.Timestamp = data.Timestamp.Unix()
```

Comment:
why don't do `Unix()` also when zero?

Reply:
It is a very negative number. The beginning of `time.Time` is far before 1970.

Reply:
ouch, TIL